requirements.txt

```diff
@@ -2,4 +2,6 @@ Keras==2.3.1
 matplotlib==3.1.3
 numpy==1.18.1
 scipy==1.4.1
-scikit-learn==0.22.1
\ No newline at end of file
+scikit-learn==0.22.1
+tensorflow==2.0.0
```

Comment:
I would use the latest stable tensorflow version `2.1.0`
I tested it with a fresh conda environnment install, it works 👍